guides/release-process.md

```diff
@@ -148,26 +172,21 @@ In the following instructions, "X.Y.Z" is used to denote the [next version of Cy
     git push origin vX.Y.Z
     ```
 
-18. Submit, get approvals on, and merge a new PR that merges `develop` to `master`
-
-19. Inside of [cypress-io/release-automations][release-automations]:
-    - Publish GitHub release to [cypress-io/cypress/releases](https://github.com/cypress-io/cypress/releases) using package `set-releases`:
+19. Submit, get approvals on, and merge a new PR that merges `develop` to `master`. **Important**: make sure to use a merge commit, not a squash merge.
 
-        ```shell
-        cd packages/set-releases && npm run release-log -- --version X.Y.Z
-        ```
+20. Create a new [GitHub release](https://github.com/cypress-io/cypress/releases). Choose the tag you created previously and add contents to match previous releases.
 
-    - Add a comment to each GH issue that has been resolved with the new published version using package `issues-in-release`:
+21. Inside of [cypress-io/release-automations][release-automations], run the following to add a comment to each GH issue that has been resolved with the new published version:
 
-        ```shell
-        cd packages/issues-in-release && npm run do:comment -- --release X.Y.Z
-        ```
+    ```shell
+    cd packages/issues-in-release && npm run do:comment -- --release X.Y.Z
+    ```
 
-    - Confirm there are no issues with the label [stage: pending release](https://github.com/cypress-io/cypress/issues?q=label%3A%22stage%3A+pending+release%22+is%3Aclosed) left
+22. Confirm there are no issues with the label [stage: pending release](https://github.com/cypress-io/cypress/issues?q=label%3A%22stage%3A+pending+release%22+is%3Aclosed) left
```

Comment:
This honestly might be a good "first" step when verifying everything has been tagged for the release and wasn't missed. 🤷🏻‍♀️

Reply:
This is covered as a first step in step #2. I think it's good to have it as a post-release step as well as it can be easy to miss something initially when there are 30 issues to go through before the release. I caught a couple today by checking the pending release tag after the release.